earthpy/spatial.py

```diff
@@ -370,6 +370,107 @@ def crop_image(raster, geoms, all_touched=True):
     return out_image, out_meta
 
 
+def crop_all(
+    band_paths,
+    output_dir,
+    geoms,
+    overwrite=False,
+    all_touched=True,
+    verbose=True,
+):
+    """Takes a list of rasters and a boundary, and crops them efficiently.
+
+    Parameters
+    ----------
+    band_paths : list of file paths
+        List of paths of bands that are to be cropped.
+    output_dir : list or string
+        If a list is provided, then the output files from this function will be written to
+        each path in the list. This allows for customization of file names.
+        If a string is provided, then the output files from the function will be written
+        to that directory with the name of the original file plus a suffix of "_crop".
+    geoms : geopandas geodataframe or list of polygons
+        The spatial polygon boundaries in GeoJSON-like dict format
+        to be used to crop the image. All data outside of the polygon
+        boundaries will be set to nodata and/or removed from the image.
+    overwrite : bool (default=False)
+        Disallows files to be overwritten if they exist already. Can be changed so that files
```

Comment:
If False, will not overwrite existing files. If true, existing files will be overwritten